server/src/routes/recipes.route.js

```diff
@@ -1,9 +1,23 @@
 import express from 'express';
+import multer from 'multer';
 import * as recipesController from '../controllers/recipes.controller';
 
 const router = express.Router();
 
-router.get('/latest/:count', recipesController.getLatest);
+const storage = multer.diskStorage({

```

Comment:
ezt tegyüt át egy szűkebb másik scopeba vagy fileba és onnan importáljuk be ide. Ez csak egyetlen egy route-hoz tartozik jelenleg

Reply:
igazából ez egy middleware